### PR TITLE
:rocket: Add Quarkus framework :coffee: 

### PR DESCRIPTION
**Pull request by adriens**
Related issue : 

- https://github.com/mingrammer/diagrams/issues/879

### CODE REVIEW

File: diagrams/programming/framework.py [hunk 1]
```diff
@@ -71,6 +71,8 @@ class Svelte(_Framework):
 class Vue(_Framework):
     _icon = "vue.png"
 
+class Vue(_Framework):
```
Comment (gastaldi) [inline diagrams/programming/framework.py:74]: Shouldn't it be:

```suggestion
class Quarkus(_Framework):
```

File: diagrams/programming/framework.py [hunk 2]
```diff
@@ -71,6 +71,8 @@ class Svelte(_Framework):
 class Vue(_Framework):
     _icon = "vue.png"
 
+class Quarkus(_Framework):
```
Comment (gastaldi) [inline diagrams/programming/framework.py:74]: Also shouldn’t classes be ordered alphabetically?

Comment (adriens) [reply diagrams/programming/framework.py:74]: I'm fixing this :zap:

File: resources/programming/framework/quarkus.png
Comment (ziegenberg) [inline resources/programming/framework/quarkus.png:1]: The image should probably have an alpha channel and the white background should be turned into alpha.

**Comment by fabriziofortino** [comment]
see #934

**Comment by gabriel-tessier** [comment]
@fabriziofortino thanks for the notice and the PR.
I will close as the #934 is already merged.

**Pull request closed without merging**